source/_components/insteon_plm.markdown

```diff
@@ -10,34 +10,70 @@ footer: true
 logo: insteon.png
 ha_category: Hub
 ha_iot_class: "Local Push"
-ha_version: 0.39
+ha_version: 0.65
```

Comment:
Don't change this. This version is when the component/platform was originally added.

Reply:
Ah, sorry did not understand that. I will change it back.